setup/setupEnvironment.sh

```diff
@@ -238,6 +238,20 @@ cat << EOF >> ${DELETE_SCRIPT}
 echo "Deleting API Gateway"
 aws apigateway delete-rest-api --rest-api-id ${GATEWAY_ID}
 
+echo "Deleting Cloudwatch logs for the Lambda functions"
```

Comment:
I'm not too comfortable merging this -- there might be a collision here with other resources (maybe they use the same 'date' value for creation purposes.